CONTRIBUTING.md

```diff
@@ -0,0 +1,65 @@
+# Contributing to the Openshift Builder
+
+The OpenShift Builder is the component that drives [OpenShift Builds](https://docs.okd.io/latest/dev_guide/builds/index.html),
+leveraging [Buildah](https://github.com/containers/buildah) and [Source-To-Image](https://github.com/openshift/source-to-image)
+to turn source code into images that can be deployed on Kubernetes.
+
+## Getting the code
+
+To get started, [fork](https://help.github.com/articles/fork-a-repo) the [openshift/builder](https://github.com/openshift/builder) repo.
+
+## Developing
+
+### Testing on an OpenShift Cluster
+
+The easiest way to test your changes is to launch an OpenShift 4.x cluster.
+First, go to [try.openshift.com](https://try.openshift.com) to obtain a pull secret and download the installer.
+Follow the instructions to launch a cluster on AWS.
+
+If you want the latest OpenShift installer and `oc` client, go to the [openshift-release](https://openshift-release.svc.ci.openshift.org/) 
+page, select the channel and build you wish to install, and download the respective `oc` and `openshift-installer` binaries.
+
```

Comment:
@gabemontero @coreydaley added a section on grabbing binaries from openshift-release, per Clayton's email on aos-devel.

Reply:
I feel like the "channel" wording here is a bit ambiguous on that page.  Is there something else we could call it?  I also only see a "Download the installer" link, nothing for the oc binary?

Reply:
The "Download the installer" link also includes tar-balls for oc (openshift-client)

Reply:
@coreydaley clarified what the "channel" means.

Learned the hard way today that you need a pull secret to our CI to get the `.ci` installs to work.